apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -116,6 +115,58 @@ const standardsLink = (locale: Locales, chunks: ReactNode) => (
   </Link>
 );
 
+const gcDigitalTalentLink = (locale: Locales, chunks: ReactNode) => (
+  <Link
+    external
+    href={
```

Comment:
Have a look at `apps/web/src/hooks/useRoutes.ts` for the routes you need in other instances.

Try
```
href={paths.register()}
```

Instead of 

```
href={
      locale === "en"
        ? "https://talent.canada.ca/en/register-info"
        : "https://talent.canada.ca/fr/register-info"
    }
```